other/materials_designer/create_interface_with_relaxation_ase_emt.ipynb

```diff
@@ -4,436 +4,197 @@
    "cell_type": "markdown",
```

Comment:
<p>Remove "1" from the first sentence after title</p><p> </p><p>Remove "The relaxation is performed to minimize the energy of the interface."</p><p> </p><p>In the introduction notebook explain how the data is saved in "cache" folder and put a link here to that explanation in Usage item 1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/126/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Ah. Link!</p>